app/src/main/java/grinnell/appdev/edu/lyles/preferences/FavoritesManager.java

```diff
@@ -0,0 +1,126 @@
+package grinnell.appdev.edu.lyles.preferences;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.util.Log;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.concurrent.ExecutionException;
+
+import grinnell.appdev.edu.lyles.AsyncRetrieval;
+import grinnell.appdev.edu.lyles.MenuItem;
+
+/**
+ * Created by Shelby on 2/3/2017.
+ * A class which uses SharedPreferences to keep track of each user's favorites in each category
+ */
+
+public class FavoritesManager {
+
+    private static final String PREFS_KEY = "Preferences";
+
+    private SharedPreferences favorites;
+    private SharedPreferences.Editor editor;
+    private ArrayList<MenuItem> allItems;
+
+    /**
+     * Creates a new favorites manager based on an Arraylist of menuItems
+     *
+     * @param context   the context of activity using the FavoritesManager
+     * @param menuItems the list of all items the FavoritesManager will keep track of
+     */
+    public FavoritesManager(Context context, ArrayList<MenuItem> menuItems) {
+
+        allItems = menuItems;
+        favorites = context.getSharedPreferences(PREFS_KEY, Context.MODE_PRIVATE);
```

Comment:
Let's use static imports for things like `Context.MODE_PRIVATE`. To do that, we shall have an import statement like this `import static android.content.Context.MODE_PRIVATE` and just call `MODE_PRIVATE` here.